astropy_helpers/sphinx/conf.py

```diff
@@ -135,15 +135,11 @@ def check_sphinx_version(expected_version):
     'sphinx.ext.coverage',
     'sphinx.ext.inheritance_diagram',
     'sphinx.ext.viewcode',
-    'astropy_helpers.sphinx.ext.numpydoc',
-    'astropy_helpers.sphinx.ext.astropyautosummary',
-    'astropy_helpers.sphinx.ext.autodoc_enhancements',
-    'astropy_helpers.sphinx.ext.automodsumm',
```

Comment:
The sphinx-automodapi extension officially only recommends adding ``automodapi`` and optionally ``smart_resolver``, so in that sense it's 'public API' to not include automodsumm and  autodoc_enhancements. But I'm happy to add this back if you really would prefer to see them here.